cilium/README.md

```diff
@@ -60,12 +62,12 @@ To configure this check for an Agent running on a host:
         instances:
         
             ## @param use_openmetrics - boolean - optional - default: false
-            ## Use the latest OpenMetrics V2 implementation for more features and better performance.
+            ## Use the latest OpenMetrics implementation for more features and better performance.
             ##
             ## Note: To see the configuration options for the legacy OpenMetrics implementation (Agent 7.33 or older),
-            ## https://github.com/DataDog/integrations-core/blob/7.33.x/cilium/datadog_checks/cilium/data/conf.yaml.example
+            ## see https://github.com/DataDog/integrations-core/blob/7.33.x/cilium/datadog_checks/cilium/data/conf.yaml.example
```

Comment:
Note that these come from: https://github.com/DataDog/integrations-core/blob/b4551e32383dc457a5810152599d21bfc99178cb/cilium/datadog_checks/cilium/data/conf.yaml.example#L48-L52

So changing the spec.yaml to match and syncing config would probably be the right thing to do:
https://github.com/DataDog/integrations-core/blob/b4551e32383dc457a5810152599d21bfc99178cb/cilium/assets/configuration/spec.yaml#L10-L15

Reply:
Thanks for all of the fixes, @alopezz!